examples/non-monorepo/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "non-monorepo",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --turbo",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@types/node": "18.11.9",
+    "@types/react": "18.0.25",
+    "@types/react-dom": "^18.0.6",
+    "eslint": "8.27.0",
+    "eslint-config-next": "13.0.2",
+    "next": "13.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "typescript": "4.8.4"
+  },
+  "devDependencies": {
+    "turbo": "^1.6.3"
+  }
```

Comment:
Turbo complains if we don't fill this out. I also want to not that if you set it up with `pnpm` you get build errors:
```
olszewski@chriss-mbp non-monorepo % pnpm turbo build                                                    
• Running build                                                                                         
• Remote caching disabled                                                                               
build: cache miss, executing 777b5f29c51dced0                                                           
build:                                                                                                  
build: > non-monorepo@0.1.0 build /private/tmp/non-monorepo                                             
build: > next build                                                                                     
build:                                                                                                  
build: /private/tmp/non-monorepo/.pnp.cjs:11864                                                         
build:     throw firstError;                                                                            
build:     ^                                                                                            
build:                                                                                                  
build: Error: Your application tried to access @next/env, but it isn't declared in your dependencies; th
is makes the require call ambiguous and unsound.                                                        
build:                                                                                                  
build: Required package: @next/env                                                                      
build: Required by: /private/tmp/non-monorepo/node_modules/.pnpm/next@13.0.2_biqbaboplfbrettd7655fr4n2y/
node_modules/next/dist/build/                                                                           
build:                                                                                                  
build: Require stack:                                                                                   
build: - /private/tmp/non-monorepo/node_modules/.pnpm/next@13.0.2_biqbaboplfbrettd7655fr4n2y/node_module
s/next/dist/build/index.js                                                                              
build: - /private/tmp/non-monorepo/node_modules/.pnpm/next@13.0.2_biqbaboplfbrettd7655fr4n2y/node_module
s/next/dist/cli/next-build.js                                                                           
build: - /private/tmp/non-monorepo/node_modules/.pnpm/next@13.0.2_biqbaboplfbrettd7655fr4n2y/node_module
s/next/dist/lib/commands.js                                                                             
build: - /private/tmp/non-monorepo/node_modules/.pnpm/next@13.0.2_biqbaboplfbrettd7655fr4n2y/node_module
s/next/dist/bin/next                                                                                    
build:     at Function.require$$0.Module._resolveFilename (/private/tmp/non-monorepo/.pnp.cjs:11863:13) 
build:     at Function.require$$0.Module._load (/private/tmp/non-monorepo/.pnp.cjs:11713:42)            
build:     at Module.require (node:internal/modules/cjs/loader:1005:19)                                 
build:     at require (node:internal/modules/cjs/helpers:102:18)                                        
build:     at Object.<anonymous> (/private/tmp/non-monorepo/node_modules/.pnpm/next@13.0.2_biqbaboplfbre
ttd7655fr4n2y/node_modules/next/dist/build/index.js:6:12)                                               
build:     at Module._compile (node:internal/modules/cjs/loader:1105:14)                                
build:     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)                  
build:     at Object.require$$0.Module._extensions..js (/private/tmp/non-monorepo/.pnp.cjs:11907:33)    
build:     at Object.require$$0.Module._extensions..js (/private/tmp/non-monorepo/.pnp.cjs:11907:33)    
build:     at Module.load (node:internal/modules/cjs/loader:981:32)                                     
build:  ELIFECYCLE  Command failed with exit code 1.                                                    
build: ERROR: command finished with error: command (/private/tmp/non-monorepo) pnpm run build exited (1)
```

```suggestion
  },
  "packageManager": "yarn@1.22.19"
```

Reply:
We don't include these in examples so they can work with any package manager out of the box (and be used by our examples scripts)